### ISSUE TITLE
Drawer is unusable on a standalone installed PWA

### ISSUE THREAD

**Issue by smhutch** [emilkowalski/vaul#505]
`Drawer` content is unusable when:

1. The app has a `manifest.json` with `"display": "standalone",`
2. The user installs the app on iOS
3. A sheet is rendered below the fold on a page (interestingly it works fine if you are scrolled to the top)

## Recording of issue

https://github.com/user-attachments/assets/623a1ece-b120-4e5b-8255-e2c3fef6e585

In this recording I:

1. Have installed the app on iOS
2. Open the `Drawer` at the top of the page. I can interact with it as normal and click on the button to show the alert.
3. Scroll to the bottom, and open the problematic `Drawer` (same component as above).
4. First issue is that the body scrolls back to the top (you can see that behind the drawer)
5. Second issue is that if I interact with the drawer in any way it instantly closes

## Minimal repro

1. [App](https://vaul-bug-report.f8n.dev/) showing the bug
6. [Repo with minimal repro](https://github.com/smhutch/vaul-bug-report)

**Comment by smhutch** [comment]
@emilkowalski what's the purpose of [this condition](https://github.com/emilkowalski/vaul/blame/17a60c592eeea89584d3a5a9f914a31b4eaa7b2d/src/use-position-fixed.ts#L130-L132)? I found that removing it fixed the issues we have been seeing on a standalone PWA. I assume it has some purpose though.

**Comment by hiroingk** [comment]
The same issue is occurring in iOS app WebViews, not just PWAs.
As a workaround, we resolved it by setting the WebView's User Agent to match Safari's UA as shown in the link below:
https://www.whatismybrowser.com/guides/the-latest-user-agent/ios
However, we hope it can be made to work without depending on the UA.

**Comment by Lovresqu** [comment]
> at removing it fixed the issues we

@smhutch How did you remove it?

**Comment by kailoklum** [comment]
yep can confirm this behaviour on iOS as well using React Native's Webview. Temporary workaround by changing the user agent for webview works.

**Comment by nano72mkn** [comment]
Setting `repositionInputs` to false solved it for me!

**Comment by smhutch** [comment]
> @smhutch How did you remove it?

@Lovresqu I forked the library locally as a test. I didn't put this change into production, was just debugging to try to understand the problem a bit more.

**Comment by alishahlakhani** [comment]
I got vaul implementation from shadcn's drawer component. Got it to work by just adding `modal={false}` however this removes the fade dark background from behind which is still okay for what I'm building.

https://github.com/user-attachments/assets/cd6e1c9e-182e-4026-bda4-8ae02b00e39d

**Comment by mithi** [comment]
I'm experiencing the same issue with my PWA on iOS
